src/js/components/FileInput/FileInput.js

```diff
@@ -461,29 +462,42 @@ const FileInput = forwardRef(
                 justify="between"
                 direction="row"
                 align="center"
+                style={{ zIndex: 0 }} // make the tip appear on file name hover
               >
                 {renderFile ? (
                   renderFile(file)
                 ) : (
-                  <Box
-                    {...theme.fileInput.label}
-                    gap="xsmall"
-                    align="center"
-                    direction="row"
+                  <Tip
+                    content={file.name}
+                    dropProps={{
```

Comment:
I dont think we need any of these `dropProps` can we just let the drop be styled from the theme with whats already there